mlflow/llama_index/__init__.py

```diff
@@ -321,6 +330,15 @@ def log_model(
         Please refer to the `Models From Code Guide <https://www.mlflow.org/docs/latest/model/models-from-code.html>`_
         for more information.
 
+    .. note::
+
+        When logging a model, MLflow will automatically saves the state of the ``Settings``
+        object so that you can use the same settings at inference time. However, please
```

Comment:
```suggestion
        When logging a model, MLflow will automatically save the ``Settings``
        object as JSON so that you can use the same settings at inference time. However, please
```

Can we remove `the state of`? It sounds like MLflow saves `settings.state`.